src/components/sellerPages/profile/dashboard/infoSection/EditProfileForm.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useEffect } from 'react';
+import { Modal, Input, Form, Typography } from 'antd';
+import AutoComplete from 'react-google-autocomplete';
+
+const EditProfileForm = ({ visible, fields, onSubmit, onCancel }) => {
+  const [address, setAddress] = useState('');
+  const [addressValue, setAddressValue] = useState('');
+  const { Text } = Typography;
+
+  useEffect(() => {
+    setAddress(fields.length > 0 ? fields[2].value : '');
+    setAddressValue(fields.length > 0 ? fields[2].value : '');
+  }, [fields]);
+
+  const [form] = Form.useForm();
+
+  const validatePhone = (rule, value, callback) => {
+    const regex = new RegExp(/\([0-9]{3}\)\s[0-9]{3}-[0-9]{4}/);
```

Comment:
this phoner validation is really nice, that way we keep consistency in the data submitted 😃